py-polars/polars/io.py

```diff
@@ -705,6 +705,68 @@ def scan_parquet(
     )
 
 
+def scan_json(
```

Comment:
General question about `json` vs. `ndjson`, will this function only work on `.ndjson` files? If so, that should be documented somewhere.

Reply:
Good call. I'll add to the docstring that this only supports ndjson as of now.

Reply:
@matteosantama i guess I already added this to the docstring.